scripts/sonic_py.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+
+"""
+    Sonic_py is the library that provides APIs to access database.
+"""
+
+import redis
+
+
+REDIS_UNIX_SOCKET = "/var/run/redis/redis.sock"
+CNT_DB = 2
+COUNTER_PORT_NAME_MAP = "COUNTERS_PORT_NAME_MAP"
+
+class sonic_py(object):
```

Comment:
I think we should use that. It seems that the library is a little bit out of date, but we could still pick it up. For the other tool, the 'aclshow' in https://github.com/Azure/sonic-utilities/pull/18, I think we also need to use that.